### ISSUE TITLE
Non "Mono" PragmataPro not showing up in gvim.exe

### ISSUE THREAD

**Issue by jordwalke** [fabrizioschiavi/pragmatapro#116]
Using the latest 8.1 gvim.exe release, PragmataPro fonts (not the Mono versions) don't even show up in the gvim.exe font picker. I believe I read somewhere that gvim.exe requires that *every* character be marked as monospace for the whole font to be considered fixed width and thus become available in the font picker. Without being available in the font picker, the font is not usable on gvim.exe.

The monospace forms work great (and ligatures too with the right gvim settings).

**Comment by Porges** [comment]
@jordwalke how did you get ligatures working?

**Comment by jordwalke** [comment]
@Porges there's some render options you can tweak in gvim.exe. Tons of combinations, and you have to try them until it looks good and performs well.

**Comment by jordwalke** [comment]
@fabrizioschiavi 

Couldn't you make it so that the non-monospaced fonts are considered monospace-but-with-ligatures? So imagine if you have a text file with one of those double-wide glyphs, followed by a space. You could create ligatures for every "double-wide-followed-by-space" so that it renders the double-wide glyph as a ligature. That way, it will work in far more editors.

**Comment by fabrizioschiavi** [comment]
@jordwalke 
It seems you have found the right idea and it seems works. I need to test a lot before to say "victory" anyway but thank you!

**Comment by jordwalke** [comment]
@fabrizioschiavi wow, that is exciting! Let me know if you want anyone to test this on Windows gvim.exe etc (message me on Twitter).

**Comment by Porges** [comment]
Those look like it's falling back to another font (Segoe Ui Emoji).

**Comment by jordwalke** [comment]
@porges yes, so if it's accomplishing double wide rendering in a font marked as fixed width, then why cannot PragmataPro?

**Comment by r03** [comment]
I can't even select the mono version in gvim windows. 

With these settings I get the font working: (without ligatures)

		set renderoptions=type:directx
		set encoding=utf-8
		set guifont=PragmataPro\ Mono\ Liga:h12

Without the first 2 settings I get the font in italic.

This is the vim I installed: 
https://github.com/vim/vim-win32-installer/releases/download/v8.1.0578/gvim_8.1.0578_x64-mui2.exe

font version 0.827
